.packaging/Dockerfile.gentoo

```diff
@@ -0,0 +1,29 @@
+FROM gentoo/stage3
+
+RUN echo 'FEATURES="-ipc-sandbox -network-sandbox -pid-sandbox -sandbox -usersandbox -usersync -userfetch -userpriv"' >> /etc/portage/make.conf
+
+
+COPY $CUSTOM_CERT /usr/local/share/ca-certificates/
+RUN update-ca-certificates
+
+
+RUN emerge-webrsync --no-pgp-verify \
+    && emerge --update --deep --newuse @world \
+    && emerge app-portage/gentoolkit dev-vcs/git sudo vim app-eselect/eselect-python dev-python/pip cmake dev-build/ninja clang
+
+RUN echo 'export PKGDIR=/data/packages' >> ~/.bashrc
```

Comment:
These should all be a single line, like the `emerge-webrsync` invocation above.

Reply:
done